ispyb/model/__future__.py

```diff
@@ -472,6 +479,53 @@ def _get_detector(self):
         self._data = cursor.fetchone()
 
 
+def _get_sample_group(self):
```

Comment:
Can we either add a JIRA ticket or add the line
```suggestion
def _get_sample_group(self):
    # not yet requested
```
This allows us to see at a glance what the status of all the `future` methods is